src/components/ScholarshipDetailCard.jsx

```diff
@@ -17,6 +17,10 @@ import ScholarshipAmount from '../types/ScholarshipAmount';
 import { BRAND_NAME } from '../config/constants';
 
 const useStyles = makeStyles((theme) => ({
+  detailSection: {
+    backgroundColor: '#fff',
```

Comment:
Based on the [palette colors](https://material-ui.com/customization/palette/#dark-mode)
```suggestion
    backgroundColor: theme.palette.background.paper,
```

Reply:
ended up going with the Card component instead